.github/workflows/ci.yaml

```diff
@@ -183,3 +182,72 @@ jobs:
           emulator-options: -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
           disable-animations: true
           script: ./gradlew connectedCheck -Pandroid.testInstrumentationRunnerArguments.class=org.ZingoLabs.Zingo.IntegrationTestSuite
+  
+  e2e_test:
+    name: End-to-End Test
+    runs-on: macos-12
+    strategy:
+      matrix:
+        include:
+          - arch : x86
+            api-level: 29
+            jniLibs_dir: x86
```

Comment:
if you see the integration test job, jniLibs_dir is actually not necessary. you can delete this line and replace all matrix.jniLibs_dir with matrix.arch.

this was a hangover from the ARM stuff i was doing but this wasn't needed in the end